libs/features/mas/mocks/aem.js

```diff
@@ -3,12 +3,12 @@ export async function withAem(originalFetch) {
         if (/cf\/fragments\/search/.test(pathname)) {
             // TODO add conditional use case.
             return originalFetch(
-                '/test/mocks/sites/cf/fragments/search/default.json',
+                '/test/mocks/sites/cf/fragments/search/authorPayload.json',
```

Comment:
It is implied that the API and response is for author, since Freyja/publish doesn't support this.
This name might be misleading like one can think that there may be also a publishPayload for search api.
how about `searchResponse.json` ?  Neutral and explicit.

Reply:
there will be 'edgePayload.json' or something like this to test Freya